htdocs/js/jquery.imageshrink.js

```diff
@@ -18,19 +25,19 @@ jQuery(function($) {
     // Only shrink casual pics; leave artisanal HTML alone. Can't predict
     // everything, but 99% of the time that's a <div style="..."> or a table.
     function protectTheCodes(container) {
-        // Expanded cut tags have style='display: block', but aren't decorations.
-        // Feeds (.journal-type-Y) always make a mess, so no mercy.
-        var exemptImages = container.querySelectorAll(
-            '.journal-type-P .entry-content div[style]:not(.cuttag-open) img, ' +
-            '.journal-type-P .entry-content table img, ' +
-            '.journal-type-C .entry-content div[style]:not(.cuttag-open) img, ' +
-            '.journal-type-C .entry-content table img, ' +
-            '.comment-content div[style] img, ' +
-            '.comment-content table img'
-        );
+        let queryStrings = imgHolders.map(function(holderName) {
```

Comment:
The selector string this builds is completely static, just like the "imgHolders + img" one above, so I suggest pulling it out of this more active code path and constructing it only once in that outer scope. (And, pre-baking the `.join(', ')` from below into it at the same time, so the pre-computed value is just one string.) 

Also, repeat same notes from above about "let" and calling em "selectors".